spec/unit/lib/temporal/client_spec.rb

```diff
@@ -310,6 +310,18 @@ def expect_signal_with_start(expected_arguments, expected_signal_argument)
     end
   end
 
+  describe '#describe_namespace' do
+    before { allow(connection).to receive(:describe_namespace).and_return(Temporal::Api::WorkflowService::V1::DescribeNamespaceResponse.new) }
+    
+    it 'passes the namespace to the connection' do
+      result = subject.describe_namespace('new-namespace')
+
+      expect(connection)
+        .to have_received(:describe_namespace)
+        .with(name: 'new-namespace')
+      expect(result).to be_an_instance_of(Temporal::Api::WorkflowService::V1::DescribeNamespaceResponse)
```

Comment:
nit: This assertion doesn't add any value since it's only making sure Spec works as expected